easybuild/easyblocks/g/gcc.py

```diff
@@ -88,13 +103,18 @@ def extra_options():
             'withlto': [True, "Enable LTO support", CUSTOM],
             'withppl': [False, "Build GCC with PPL support", CUSTOM],
             'withnvptx': [False, "Build GCC with NVPTX offload support", CUSTOM],
+            'withamdgcn': [False, "Build GCC with AMD GCN offload support", CUSTOM],
         }
         return ConfigureMake.extra_options(extra_vars)
 
     def __init__(self, *args, **kwargs):
         super(EB_GCC, self).__init__(*args, **kwargs)
 
         self.stagedbuild = False
+        # Each build iteration is related to a specific build, first build host compiler, then potentially build NVPTX
+        # offloading and/or AMD GCN offloading
+        # NOTE: we are using a dictionary here, intentionally, and not a list to ensure that indexing is not off by one
+        self.build_stages = {0: HOST_COMPILER}
```

Comment:
Not sure I like this... How could we have a off-by-one error if all we have is `0`, `1`, `2` as keys in `self. build_stages`?!

Reply:
I had it like this before I added the `HOST_COMPILER` variable, I will change it to a list